### DISCUSSION TITLE
A way to lock some Stream Deck buttons when when my session is locked

### DISCUSSION THREAD

**Discussion by sftim** [streamdeck-linux-gui/streamdeck-linux-gui#15] [Ideas / Feature Requests]
As a security-conscious user, I want the Stream Deck to lock (dim the screen and not respond to button pushes) whenever my session is locked.

Relevant to issues https://github.com/timothycrosley/streamdeck-ui/issues/4 and https://github.com/timothycrosley/streamdeck-ui/issues/7

Previously https://github.com/timothycrosley/streamdeck-ui/issues/46

**Comment by RemcoSchrijver** [comment]
#23 is submitted so it seems it is on the radar now.

**Comment by sftim** [comment]
By the way, I personally like that some buttons stay active even when my logind session isn't active. For example, I use a button to sleep the PC, and would miss that a little if it disappeared when the session is locked.

I realize that the first pass is probably to blank the Stream Deck for a locked session.

**Reply by celgit** [reply]
Maybe a select-box that user can check if they want the button to work when computer is locked, default is off.
